### ISSUE TITLE
Feature request: Test accessibility of search form using Cypress

### ISSUE THREAD

**Issue by dmundra** [CivicActions/accessibility#409]
**Is your feature request related to a problem? Please describe.**
Pa11y-ci will test the search page but not the search results as it is dynamic. We might be missing accessibility issues in the search results that we have to check manually.

**For whom is this feature?**
As QA Quincy, I want to test the search form and results automatically, so that I do not have to do it manually.

**Describe the solution you'd like**
Use Cypress and write a test that interacts with the search form and then tests the results for accessibility issues.

**Describe alternatives you've considered**
See if Pa11y-ci could also test the search form after interacting with it.
